test/fixtures/rules/std_rule_cases/L003.yml

```diff
@@ -590,7 +632,7 @@ test_tsql_function:
             ) + '12' + CAST(24 + DATEPART(day, @DATE) AS char(2))) + 1;
     --Special case Dec 29-31 may belong to the next year
         IF ((DATEPART(mm, @DATE) = 12)
-        AND ((DATEPART(dd, @DATE) - DATEPART(dw, @DATE)) >= 28))
```

Comment:
This single test is the only thing changed in terms of test - I noticed both now and before the final state of this query is based on the input state (which seems super weird to me). I found this test hard to reason about what is correct in terms of indent for this.